### ISSUE TITLE
Mobile App COVID Cases Hexagonal Map View

### ISSUE THREAD

**Issue by ggoslin** [cmu-delphi/www-covidcast#1181]
**Steps to Reproduce the Bug:**

1. go to https://delphi.cmu.edu/covidcast/indicator/?sensor=indicator-combination-confirmed_7dav_incidence_prop
2. open in your phone browser
3. examine the United States choropleth map

**Actual Behavior:**
*Only the color indicating the number of COVID cases for each state is shown. Is this intentional?

**Expected behavior**
*I expected there to be graphs for each state showing the COVID case number in the recent few days

**Screenshots**

![218](https://user-images.githubusercontent.com/68612840/171505404-97b4cb86-f470-42fa-b6d6-5758c7608540.jpg)

**Rating Scale (1 is minor, 7 is severe):** 3. The big ideas are still represented in the map, so the user knows generally which states have high cases, but the user is unable to see or compare any previous trends quickly.

**Context**

- Browser: Safari on iPhone XR

**Comment by krivard** [comment]
@sgratzl do you remember if this was on purpose, for screen size reasons?

**Comment by sgratzl** [comment]
yeah because of screen size the small sub charts are not showing up on mobile. There is just no space to show the label in a readable fashion along with the charts.
